client/src/renderer/components/Clipboard/index.jsx

```diff
@@ -0,0 +1,66 @@
+/** @format */
+
+import "./styles.scss";
+import { Component, h } from 'preact';
+import clipboard from "electron-clipboard-extended";
+
+export default class Clipboard extends Component {
```

Comment:
Please give all components and custom functions some documentation. For examples of this, see other components within this repo, although we haven't done a great job so far. Please follow the tag convention defined by [esdoc](https://esdoc.org/manual/tags.html).

Keep the first line of the documentation brief and to the point. You can go into more detail in the next section, including examples and links to relevant content. Remember, you're writing code for three audiences:

 1. Yourself
 2. The computer
 3. Anyone else who looks at the code in the future, which could be you!

So it's important to document your code for them, and to follow conventions wherever possible.

```suggestion
/**
 * Tracks changes to the clipboard, and shows them in a list.
 * @extends Component
 */
export default class Clipboard extends Component {
```